great_expectations/validator/validator.py

```diff
@@ -79,7 +79,7 @@
         "Unable to load pandas; install optional pandas dependency for support."
     )
 
-MAX_METRIC_COMPUTATION_RETRIES = 3
+MAX_METRIC_COMPUTATION_RETRIES: int = 3
```

Comment:
Does this need to be a global constant? Couldn't we just put 3 as a default arg in the decorator?

Reply:
@cdkini Good idea.  In next commit.